samples/Calling/src/app/views/CallScreen.tsx

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import React, { useEffect, useState } from 'react';
+import { Spinner } from '@fluentui/react';
+import { GroupCallLocator, TeamsMeetingLinkLocator } from '@azure/communication-calling';
+import { CallAdapter, CallComposite, createAzureCommunicationCallAdapter } from '@azure/communication-react';
+import { CommunicationUserIdentifier } from '@azure/communication-common';
+import { refreshTokenAsync } from '../utils/refreshToken';
+import { useSwitchableFluentTheme } from '../theming/SwitchableFluentThemeProvider';
+
+export interface CallScreenProps {
+  token: string;
+  userId: CommunicationUserIdentifier;
+  callLocator: GroupCallLocator | TeamsMeetingLinkLocator;
+  displayName: string;
+  onCallEnded: () => void;
+  onCallError: (e: Error) => void;
+}
+
+export const CallScreen = (props: CallScreenProps): JSX.Element => {
+  const { token, userId, callLocator, displayName, onCallEnded, onCallError } = props;
+  const [adapter, setAdapter] = useState<CallAdapter>();
+  const { currentTheme } = useSwitchableFluentTheme();
+
+  useEffect(() => {
+    (async () => {
+      console.log('creating adapter');
```

Comment:
just want to confirm we want to put this creating adapter log line here?

Reply:
nope nice catch, will remove